app/controllers/new_register_controller.rb

```diff
@@ -0,0 +1,56 @@
+require 'json'
+
+class NewRegisterController < ApplicationController
+
+  layout 'layouts/default/application'
+
+  def index
+  end
+
+  def select_request_or_suggest
+    if params[:subject] == 'request'
+      redirect_to new_register_request_register_path
+    else
+      redirect_to new_register_suggest_register_path
+    end
+  end
+
+  def suggest_register
+    @new_register = Spina::NewRegister.new
+  end
+
+  def request_register
+    @new_register = Spina::NewRegister.new
+  end
+
+  def thanks
```

Comment:
what is this empty method doing?

Reply:
It's to identify the view and route `thanks#new_registers`